recipes/botan/all/conanfile.py

```diff
@@ -516,6 +516,8 @@ def _is_glibc_older_than_2_25_on_linux(self):
         # https://github.com/conan-io/conan-center-index/pull/18079#issuecomment-1919206949
         # https://github.com/conan-io/conan-center-index/pull/18079#issuecomment-1919486839
 
+        if self.settings.os != 'Linux':
+            return False
         libver = platform.libc_ver()
         return (
             self.settings.os == 'Linux' and
```

Comment:
This check could now be deleted :)

Reply:
I guess so. But I'd prefer to proceed cautiously due to my limited knowledge on this subject. It's better to have it removed by a more experienced developer.